Airports/EGNM/Sector.txt

```diff
@@ -1,16 +1,16 @@
 CIRCLE_SECTORLINE:Only NMTWR:EGNM:2.5
-DISPLAY:Scottish PC East:Only NMTWR 1:Only NMTWR 2
+DISPLAY:East:Only NMTWR 1:Only NMTWR 2
 
 CIRCLE_SECTORLINE:Only NMAPP:EGNM:5
-DISPLAY:Scottish PC East:Scottish PC East:Only NMAPP
+DISPLAY:East:East:Only NMAPP
 
 CIRCLE_SECTORLINE:NMTWR:EGNM:2.5
 DISPLAY:NMAPP:NMAPP:NMTWR
 DISPLAY:NMTWR:NMAPP:NMTWR
 
 SECTORLINE:NMAPP
-DISPLAY:Scottish PC East:Scottish PC East:NMAPP
-DISPLAY:NMAPP:Scottish PC East:Scottish PC East
+DISPLAY:East:East:NMAPP
```

Comment:
Do we need to display this to East? We have the online ring displaying too, but I suppose this marks out the area of EGNM APP...